src/style.css

```diff
@@ -0,0 +1,59 @@
+body {
+  margin: 0 auto;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+  position: relative;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
```

Comment:
`space-evenly` is probably causing the problems with test, you should position your stars with margins in this example, this gives us more control